apps/web/composables/useWishlist/types.ts

```diff
@@ -17,6 +19,7 @@ export type DeleteWishlistItem = (params: DeleteWishlistItemParams) => Promise<b
 export type IsWishlistItem = (variationId: number) => boolean;
 export type InteractWithWishlist = (variationId: number, quantity: number) => Promise<void>;
 export type SetWishlistItemIds = (wishlistItemIds: string[]) => void;
+export type SetWishlistVariationIds = (wishlistItems: any[]) => void;
```

Comment:
I no longer think this method is needed, for for future reference, please never use any for types, it's just a hack so that the build works but it should have the correct type.